src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -120,6 +122,10 @@ export class MeetingManager implements AudioVideoObserver{
     this.audioVideoObservers = {};
   }
 
+  setMeetingConfigurationLogLevel = (logLevel: LogLevel) => {
+    this.meetingConfiguration.logLevel = logLevel;
+  }
```

Comment:
Rather than having an explicit API, I think it would be a bit better if we allowed it to be passed with the MeetingProvider

```
const Root = () => {
  <MeetingProvider logLevel="warn">
     <App />
  </MeetingProvider>
}
```

Something like that maybe. What do you think?

Reply:
Yes this can also be done without explicit API, will update it.